.vib/phpbb/cypress/cypress/integration/phpbb_spec.js

```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+
+it('allows to access Administrator Control Panel', () => {
+  cy.login();
+  cy.contains('Administration Control Panel').click();
+  // ACP requires double login
+  cy.get('[type="password"]').type(Cypress.env('password'));
+  cy.get('[name=login]').click();
+  cy.contains('ACP Logout');
```

Comment:
IMHO: I would simplify onlwith with "Logout"

Reply:
Unfortunately, we can't do that. The UI has two different elements: `Logout` and `ACP Logout` (regular logout from the system vs. Administration Control Panel Logout).

This is why the whole text is here 😁